app/services/team_service.rb

```diff
@@ -12,12 +12,11 @@ def invite(email, name)
     response = slack_client.admin_invite(email: email, first_name: name.to_s.scan(/^\S+/).first).symbolize_keys
     if response[:ok]
       ActivityWorker.perform_async @team.space_id, I18n.t('team_service.invited', name: name, email: email)
-      Rails.logger.info "#{@team.space.subdomain}: invited user #{email}/#{name}"
     else
       ActivityWorker.perform_async @team.space_id,
         I18n.t('team_service.invite_failed', name: name, email: email, error: response[:error].to_s.gsub('_', ' ')),
         'WARN'
-      Rails.logger.info "#{@team.space.subdomain}: error inviting #{email}/#{name} to team: #{response}"
+      Rails.logger.info "#{@team.space.subdomain}: error inviting a member to team: #{response}"
```

Comment:
any way we can log the member id  here? otherwise this will be very useless

Reply:
@FrauBienenstich well it did fix the custom log calls. but yes, still need to remove the emai/name params. which should be easier now that we have the membership id eerywhere.